base_delivery_carrier_label/models/stock_quant_package.py

```diff
@@ -12,28 +12,25 @@ class StockQuantPackage(models.Model):
     parcel_tracking_uri = fields.Char(
         help="Link to the carrier's tracking page for this package."
     )
-    total_weight = fields.Float(
-        digits="Stock Weight",
-        help="Total weight of the package in kg, including the "
-        "weight of the logistic unit.",
-    )
 
-    @api.depends("total_weight")
+    @api.depends("shipping_weight")
     def _compute_weight(self):
         """Use total_weight if defined
```

Comment:
Doc string to update as it still mention `total_weight`

Reply:
Fixed